Code/GraphMol/ForceFieldHelpers/CrystalFF/TorsionPreferences.cpp

```diff
@@ -151,12 +161,38 @@ void getExperimentalTorsions(const RDKit::ROMol &mol, CrystalFFDetails &details,
   unsigned int aid1, aid2, aid3, aid4;
   unsigned int bid2;
 
+  // exclude bonds in bridged ring systems
+  boost::dynamic_bitset<> excludedBonds(nb);
+  const RingInfo *rinfo = mol.getRingInfo();
+  const VECT_INT_VECT &bondRings = rinfo->bondRings();
+  VECT_INT_VECT_CI rii, rjj;
+  for (rii = bondRings.begin(); rii != bondRings.end(); rii++) {
+    boost::dynamic_bitset<> rs1(nb); // bitset for ring 1
+    for (unsigned int i = 0; i < rii->size(); i++) {
+      rs1[(*rii)[i]] = 1;
+    }
+    for (rjj = rii+1; rjj != bondRings.end(); rjj++) {
+      boost::dynamic_bitset<> rs2(nb); // bitset for ring 2
```

Comment:
The block of code between lines 175 and 179 would be considerably more efficient as:
```
      unsigned int nInCommon = 0;
      for (auto rjj_i : *rjj) {
        if (rs1[rjj_i]) {
          ++nInCommon;
          if (nInCommon > 1) {
            break;
          }
        }
      }
      if (nInCommon > 1) {  // more than one bond in common

```